doc/code/qml.rst

```diff
@@ -6,4 +6,3 @@ qml
 .. automodapi:: pennylane
     :no-heading:
     :include-all-objects:
-    :skip: plugin_converters, default_config, version_info, defaultdict
```

Comment:
Seems like with putting io into a folder, these are no longer exposed via `from io import *` as before so this is not needed.

Reply:
❤️